compiler/pytest.vim

```diff
@@ -15,14 +24,19 @@ endif
 let s:cpo_save = &cpo
 set cpo-=C
 
-CompilerSet makeprg=py.test\ --tb=short\ -q
-
 CompilerSet errorformat=
-  \%*[_]\ %m\ %*[_],
-  \%A%>%f:%l:\ %.%#,
-  \%-Z>%*[\ ]%m,
-  \E%m,
-  \%-G%.%#
+    \%-G%.%#FF%.%#,
```

Comment:
What is this meant to match and ignore?
Since it (appears to be) case insensitive, it removes all lines containing `ff`.